ansible/update_pw_python.yml

```diff
@@ -0,0 +1,21 @@
+---
+- hosts: all
+  tasks:
+    - include: update_pw_python.yml
```

Comment:
This is my error. I took your suggestion from a previous comment and made this and the other convenience playbook and made them into single files. This line is a remnant of the old file.